### PR TITLE
EAP7-832 Add option for all cookies over a secure channel to be marke…

### PR DESCRIPTION
**Pull request by stuartwdouglas**
…d secure

### CODE REVIEW

File: undertow/EAP7-832_Secure_Cookie.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+:icons:             font
+:keywords:          comma,separated,tags
+:idprefix:
+:idseparator:       -
+:issue-base-url:    https://issues.jboss.org
+
+== Overview
+
+The goal of this feature is to add an option to mark all cookies that are set on a secure connection with the
+secure attribute. This means that a cookie that is set over a secure connection will never be sent over an unsecure
```
Comment (jstourac) [inline undertow/EAP7-832_Secure_Cookie.adoc]: typo - `set` -> `sent`

File: undertow/EAP7-832_Secure_Cookie.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+=== Related Issues:
+
+* {issue-base-url}/UNDERTOW-1182[UNDERTOW-1182]
+
+=== Dev Contacts:
+
+* mailto:{email}[{author}]
+
+=== QE Contacts:
+
```
Comment (jstourac) [inline undertow/EAP7-832_Secure_Cookie.adoc]: feel free to put me in

File: undertow/EAP7-832_Secure_Cookie.adoc (hunk 1 above)
Comment (stuartwdouglas) [reply undertow/EAP7-832_Secure_Cookie.adoc]: Both set and sent work here, but set is less ambiguous imho as it makes it clear it is referring to set-cookie.

Comment (jstourac) [reply undertow/EAP7-832_Secure_Cookie.adoc]: Oh, of course, you are right. In this case `set` is better. Keep it that way, then :)

**Pull request merged**